sqlglot/generator.py

```diff
@@ -259,6 +263,23 @@ def columndef_sql(self, expression):
         return f"{column} {kind}{not_null}{default}{collate}{comment}{unique}{primary}{auto_increment}"
 
     def create_sql(self, expression):
+        comment = self.sql(expression, "comment")
+        if comment:
+            if self.TABLE_COMMENT_AS_OPTION:
+                comment = f"{c.COMMENT}={comment}"
+            else:
+                properties = expression.args["properties"] or exp.Properties(
+                    expressions=[]
+                )
+                properties.args["expressions"].append(
```

Comment:
this is dangerous because you are mutating properties expressions, the generator shouldn't mutate the parent tree